application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/Macro.xml

```diff
@@ -606,6 +606,10 @@ if (typeof(XWiki) == "undefined" || typeof(XWiki.widgets) == "undefined" || type
         script.remove();
       });
       content.insert('&lt;div id="formcontent"&gt;&lt;div id="forminnercontent"&gt;' + container.innerHTML + '&lt;/div&gt;&lt;/div&gt;');
+
+      //trigger xwiki:dom:updated right after loading xform in modal
```

Comment:
This comment is not really needed, since I find the code below descriptive enough.
But if you feel it is important, it could be something like `Notify that the calendar modal was added in the document content`, in order to not simply duplicate the code below